src/nvim/option.c

```diff
@@ -2632,6 +2633,10 @@ did_set_string_option(
     if (strcmp((char *)(*varp), HIGHLIGHT_INIT) != 0) {
       errmsg = e_unsupportedoption;
     }
+  } else if (varp == &p_jop) { // 'jumpoptions'
```

Comment:
linter will want 2 spaces before the inline comment.
```suggestion
  } else if (varp == &p_jop) {  // 'jumpoptions'
```

You can run the linter with `make lint`.

Reply:
Thanks.  The linter issues caused by these changes were hard to spot among the other issues.